src/internal/views/nunjucks/view-licences/macros/licence-conditions.njk

```diff
@@ -0,0 +1,102 @@
+                <summary>View your {{ condition.points[0].points.length }} abstraction points for {% if condition.points[0].points.length == 1 %}this condition{% else %}these conditions{% endif %}</summary>
+                <dl class="govuk-summary-list summary-list--nested-list">
+                  {% for abstractionPoint in condition.points[0].points  %}
+                      <div class="govuk-summary-list__row">
+                        <dd class="{% if loop.last %} summary-list__value--no-bottom-border {%else%} govuk-summary-list__value  {%endif%}">
+                          {{ point(abstractionPoint) }}
+                        </dd>
+                      </div>
+                  {% endfor %}
+                </dl>
+              </details>
+            {% endif %}
+          </dd>
+        </div>
+        {% set outerLoop = loop %}
+        {% for conditionDetail in condition.points[0].conditions %}
+        <h4 class="govuk-visually-hidden">Condition {{ loop.index }}</h4>
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key govuk-!-font-weight-regular" id="label-{{outerLoop.index}}-{{loop.index}}-purpose">
+            Purpose
+          </dt>
+          <dd class="govuk-summary-list__value" aria-labelledby="label-{{outerLoop.index}}-{{loop.index}}-purpose">
+            {{ condition.purpose }}
+          </dd>
+        </div>
+
+        {% if conditionDetail.parameter1 %}
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key govuk-!-font-weight-regular" id="label-{{outerLoop.index}}-{{loop.index}}-purpose">
+            {{ condition.parameter1Label }}
+          </dt>
+          <dd class="govuk-summary-list__value" aria-labelledby="label-{{outerLoop.index}}-{{loop.index}}-purpose">
+            {{ conditionDetail.parameter1 }}
+          </dd>
+        </div>
+        {% endif %}
+
+        {% if conditionDetail.parameter2 %}
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key govuk-!-font-weight-regular" id="label-{{outerLoop.index}}-{{loop.index}}-purpose">
+            {{ condition.parameter2Label }}
+          </dt>
+          <dd class="govuk-summary-list__value" aria-labelledby="label-{{outerLoop.index}}-{{loop.index}}-purpose">
+            {{ conditionDetail.parameter2 }}
+          </dd>
+        </div>
+        {% endif %}
+
+        {% if conditionDetail.text %}
+        <div class="govuk-summary-list__row">
+          <dt class="govuk-summary-list__key govuk-!-font-weight-regular" id="label-{{outerLoop.index}}-{{loop.index}}-purpose">
+            Other information
+          </dt>
+          <dd class="govuk-summary-list__value" aria-labelledby="label-{{outerLoop.index}}-{{loop.index}}-purpose">
+            {{ conditionDetail.text }}
+          </dd>
+        </div>
+        {% endif %}
+
+
+        {% endfor %}
+      </dl>
+    </div>
+  </div>
+  {% if loop.index != loop.last %}<br>{% endif %}
```

Comment:
I think `loop.last` is a boolean not an integer.  So maybe this should just be:
```
{% if loop.last %}<br>{% endif %}
```
or even the inline form:
```
{{ '<br>' if loop.last }}
```

Reply:
I changed it to `{% if not loop.last %}<br>{% endif %}` which is what I wanted, just wasn't sure how to do it